dev/com.ibm.ws.jsp.2.3_fat/publish/servers/jspServer/server.xml

```diff
@@ -16,7 +16,7 @@
         <feature>jsp-2.3</feature>
     </featureManager>
     
-    <logging traceSpecification="*=info=enabled:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all:HTTPChannel=all:TCPChannel=all:GenericBNF=all:com.ibm.ws.jsp*=all" maxFileSize="32" maxFiles="24" traceFormat="BASIC"/> 
+    <logging traceSpecification="*=info=enabled:JCDI=all:com.ibm.ws.webbeans*=all:org.apache.webbeans*=all:org.jboss.weld*=all:com.ibm.ws.cdi*=all:com.ibm.ws.wswebcontainer*=all:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all:HTTPChannel=all:TCPChannel=all:GenericBNF=all:com.ibm.ws.jsp*=all" maxFileSize="32" maxFiles="24" traceFormat="BASIC"/> 
```

Comment:
Why are we adding CDI traces to a server that does not enable CDI?

Reply:
I don't think we use com.ibm.ws.wswebcontainer any more

Reply:
wswebcontainer is removed

Reply:
trace is now only added to jspCdiServer